cluster/gce/gci/configure-helper.sh

```diff
@@ -900,6 +900,9 @@ function start-kube-apiserver {
 
 
   local authorization_mode="RBAC"
+  # Load existing ABAC policy files written by versions < 1.6 of this script
+  # TODO: only default to this legacy path when in upgrade mode
+  ABAC_AUTHZ_FILE="${ABAC_AUTHZ_FILE:-/etc/srv/kubernetes/abac-authz-policy.jsonl}"
```

Comment:
LGTM for GCI.

I think you'd need the same thing above the corresponding stanza in the container-linux script.
Trusty would be the same as this, but it looks like it isn't being maintained anymore...
Debian sets this up w/ salt (https://github.com/kubernetes/kubernetes/tree/master/cluster/saltbase/salt/kube-apiserver) and it appears to still be using ABAC all the time.

Those can be handled in followup PRs.